war/src/test/java/io/lumeer/engine/controller/SecurityFacadeIntegrationTest.java

```diff
@@ -106,8 +109,24 @@ public void addAndRemoveOrganizationUserRole() throws Exception {
       securityFacade.addOrganizationUserRole(org, user, roleManage);
       assertThat(securityFacade.hasOrganizationRole(org, roleManage)).isTrue();
 
+      List<Role> roles = securityFacade.getOrganizationRoles(org);
+      roles.forEach((item) -> {
+         assertThat(LumeerConst.Security.RESOURCE_ROLES.get(LumeerConst.Security.ORGANIZATION_RESOURCE).contains(item.getName())).isTrue();
+         if (item.getName().equals(roleManage)) {
+            assertThat(item.getUsers().contains(user)).isTrue();
```

Comment:
This can be made much simpler:
```java
assertThat(roles).extracting(role -> role.getName())
 .containsOnly(LumeerConst.Security.RESOURCE_ROLES.get(LumeerConst.Security.ORGANIZATION_RESOURCE));

assertThat(roles).filteredOn(role -> role.getName().equals(roleManage))
 .extracting(role -> role.getUsers().contains(user);
```

The same applies to other such blocks.